main/ControllersPresenters/InterviewLevel.java

```diff
@@ -34,22 +33,45 @@ public String getInterviewOutput(String input) throws Exception{
             return getEndOfInterviewPrompt();
         }
         if (currentInterviewMaker.getChoiceOptions().contains(input)) {
-            //check if this is the last thing the intern says
-            //if it's not:
-                // return the corresponding interviewee's next response and next choice options to the player
-            //if it is:
-                //return the interviewee's final response, and prompt player to make hiring decision.
-                //this.getHiringResponse(input);
+            //if the current InterviewIntern has not yet said their last response:
+            if (! this.currentInterviewMaker.checkInternsLastResponse(input)){
+                return GamePrompts.INTERN_RESPONSE_PROMPT +
+                        currentInterviewMaker.displayInternChoiceResponse(input) +
+                        GamePrompts.NEXT_CHOICE_PROMPT +
+                        currentInterviewMaker.getChoiceOptions();
+            }
+
+            //if the current InterviewIntern has said their last response (i.e. at the end of the interview):
+            if (this.currentInterviewMaker.checkInternsLastResponse(input)){
+                return GamePrompts.HIRE_INTERN + this.currentInterviewMaker.internToHire(input);
+            }
+
         }
         if (Objects.equals(input, "yes") || (Objects.equals(input, "no"))){
             //hired the intern/ or not
             //return "successfully/ don't hired intern"
+            if (Objects.equals(input, "yes")){
+                this.currentInterviewMaker.hireIntern();
+                return GamePrompts.HIRED_INTERN;
+            }
+            if (Objects.equals(input, "no")){
```

Comment:
This part is still a little wrong. We need to return more than a hiring notification. See discord Phase the thing I typed before.

Reply:
Since you wrote "return" here, the following lines where you check if another intern's first response is supposed to be prompted will not be returned at the same time.